packages/cli/src/Server.ts

```diff
@@ -498,6 +500,18 @@ class App {
 			// Save the workflow in DB
 			const result = await Db.collections.Workflow!.save(newWorkflowData);
 
+			const { tags } = req.body as { tags: string | undefined };
+
+			if (tags) {
+				const tagIds = tags.split(',');
+				await TagHelpers.createRelations(result.id as string, tagIds);
```

Comment:
do we need to validate here first before creating relations like in updates?
https://github.com/n8n-io/n8n/pull/1648/files#diff-e1c41b1f8cc627242377b30fbee7af28472c81abe5f4f41da1c513f264f128e2R672